### PR TITLE
Allow retrieval of connection(s) and package data flow as JSON string

### PR DESCRIPTION
**Pull request by patricker**
- Adds an optional parameter to `get_package` that allows you to retrieve the `data_flow_json` property.
 - Adds methods for retrieving connection names/details.

### CODE REVIEW

File: xplenty/xplenty_api.py [hunk 1]
```diff
@@ -249,6 +249,21 @@ class Schedule(BaseModel):
     def __repr__(self):
         return "<Schedule '{0}'>".format(self.name)
 
+class Connection(BaseModel):
+    """Xplenty Connection."""
+
+    _strs = ['name', 'url', 'username', 
+                'unique_id', 'username', 'type',
```
Comment (danmun) [inline xplenty/xplenty_api.py]: Is `username` included twice on purpose? If not, could you please remove one of them?

File: xplenty/xplenty_api.py [hunk 2]
```diff
@@ -223,7 +223,7 @@ def __repr__(self):
 class Package(BaseModel):
     """Xplenty Package."""
 
-    _strs = ['name', 'description', 'url']
+    _strs = ['name', 'description', 'url','data_flow_json']
```
Comment (danmun) [inline xplenty/xplenty_api.py]: Could you please add a space after the last comma (for consistency)?

**Review by danmun — CHANGES_REQUESTED** [review]

**Comment by patricker** [comment]
@danmun Thanks for the feedback, let me know if there are any other changes I can make.

**Comment by danmun** [comment]
@patricker Yes, one last thing, could you please bump the version from `3.0.0` to `3.1.0` in `setup.py`?

**Comment by patricker** [comment]
@danmun Thanks for the feedback. I've been on a trip, but I've updated the code now.

**Review by danmun — APPROVED** [review]

**Comment by patricker** [comment]
@danmun I don't have merge access. Can you merge this?

**Pull request closed without merging**